app/proc/telegram.go

```diff
@@ -0,0 +1,186 @@
+package proc
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"path"
+	"strings"
+	"time"
+
+	log "github.com/go-pkgz/lgr"
+	"github.com/microcosm-cc/bluemonday"
+	tb "gopkg.in/tucnak/telebot.v2"
+
+	"github.com/umputun/feed-master/app/feed"
+)
+
+const (
+	maxTelegramFileSize = 50_000_000
+)
+
+// TelegramClient client
+type TelegramClient struct {
+	Bot     *tb.Bot
+	Timeout time.Duration
+}
+
+// NewTelegramClient init telegram client
+func NewTelegramClient(token string, timeout time.Duration) (*TelegramClient, error) {
+	if timeout == 0 {
+		timeout = time.Duration(60 * 10)
```

Comment:
@sgaynetdinov It's 600 Nanosecond. From documentation I thought it should be `Time.Second * 60`, also I would recommend moving this value to constant instead of having it defined in-line.